documentation/paths/session.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  '/users/sessions': {
+    post: {
+      tags: ['Users'],
+      description: 'Create user session',
+      operationId: 'createUserSession',
+      parameters: [],
+      requestBody: {
+        content: {
+          'application/json': {
+            schema: {
+              $ref: '#/components/schemas/SessionCreationBody'
```

Comment:
Remember to use camelCase. Check all the file because there are other words that are not in camelCase